### ISSUE TITLE
Package installation problem

### ISSUE THREAD

**Issue by mdmomeni** [woltapp/blurhash-python#13]
When I run `pip install blurhash-python`
(`Python 3.7.9 [MSC v.1900 64 bit (AMD64)] on win32`)
I get these errors : 
```
Collecting blurhash-python
  Using cached blurhash-python-1.1.1.tar.gz (181 kB)
Requirement already satisfied: cffi in c:\users\raven\desktop\code\projects\web news collecter\back\venv\lib\site-packages (from blurhash-python) (1.14.6)
Requirement already satisfied: Pillow in c:\users\raven\desktop\code\projects\web news collecter\back\venv\lib\site-packages (from blurhash-python) (8.3.1)
Requirement already satisfied: six in c:\users\raven\desktop\code\projects\web news collecter\back\venv\lib\site-packages (from blurhash-python) (1.16.0)
Requirement already satisfied: pycparser in c:\users\raven\desktop\code\projects\web news collecter\back\venv\lib\site-packages (from cffi->blurhash-python) (2.20)
Building wheels for collected packages: blurhash-python
  Building wheel for blurhash-python (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: 'c:\users\raven\desktop\code\projects\web news collecter\back\venv\scripts\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"'; __file__='"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\Raven\AppData\Local\Temp\pip-wheel-e0xfuv45'
       cwd: C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\
  Complete output (62 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.7
  creating build\lib.win-amd64-3.7\blurhash
  copying src\blurhash\_version.py -> build\lib.win-amd64-3.7\blurhash
  copying src\blurhash\__init__.py -> build\lib.win-amd64-3.7\blurhash
  running build_ext
  generating cffi module 'build\\temp.win-amd64-3.7\\Release\\blurhash._functions.c'
  creating build\temp.win-amd64-3.7
  creating build\temp.win-amd64-3.7\Release
  building 'blurhash._functions' extension
  creating build\temp.win-amd64-3.7\Release\build
  creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7
  creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release
  creating build\temp.win-amd64-3.7\Release\src
  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcbuild\temp.win-amd64-3.7\Release\blurhash._functions.c /Fobuild\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release\blurhash._functions.obj
  blurhash._functions.c
  src/common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  src/common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
  src/common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  src/common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
  src/common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/encode.c /Fobuild\temp.win-amd64-3.7\Release\src/encode.obj
  encode.c
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data        
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
  src/encode.c(55): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(55): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
  src/encode.c(80): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(80): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(76): warning C4244: 'initializing': conversion from 'int' to 'float', possible loss of data
  src/encode.c(105): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(106): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(107): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
  src/encode.c(105): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
  src/encode.c(106): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
  src/encode.c(107): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
  C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/decode.c /Fobuild\temp.win-amd64-3.7\Release\src/decode.obj
  decode.c
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data        
  C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
  src/decode.c(43): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
  src/decode.c(38): warning C4267: 'initializing': conversion from 'size_t' to 'const int', possible loss of data
  src/decode.c(57): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
  src/decode.c(58): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
  src/decode.c(71): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
  src/decode.c(82): error C2057: expected constant expression
  src/decode.c(82): error C2466: cannot allocate an array of constant size 0
  src/decode.c(82): error C2133: 'colors': unknown size
  src/decode.c(114): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29910\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
  ----------------------------------------
  ERROR: Failed building wheel for blurhash-python
  Running setup.py clean for blurhash-python
Failed to build blurhash-python
Installing collected packages: blurhash-python
    Running setup.py install for blurhash-python ... error
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\raven\desktop\code\projects\web news collecter\back\venv\scripts\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"'; __file__='"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Raven\AppData\Local\Temp\pip-record-8gga2j_2\install-record.txt' --single-version-externally-managed --compile --install-headers 'c:\users\raven\desktop\code\projects\web news collecter\back\venv\include\site\python3.7\blurhash-python'
         cwd: C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\
    Complete output (62 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.7
    creating build\lib.win-amd64-3.7\blurhash
    copying src\blurhash\_version.py -> build\lib.win-amd64-3.7\blurhash
    copying src\blurhash\__init__.py -> build\lib.win-amd64-3.7\blurhash
    running build_ext
    generating cffi module 'build\\temp.win-amd64-3.7\\Release\\blurhash._functions.c'
    creating build\temp.win-amd64-3.7
    creating build\temp.win-amd64-3.7\Release
    building 'blurhash._functions' extension
    creating build\temp.win-amd64-3.7\Release\build
    creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7
    creating build\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release
    creating build\temp.win-amd64-3.7\Release\src
    C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files 
(x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcbuild\temp.win-amd64-3.7\Release\blurhash._functions.c /Fobuild\temp.win-amd64-3.7\Release\build\temp.win-amd64-3.7\Release\blurhash._functions.obj
    blurhash._functions.c
    src/common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    src/common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
    src/common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    src/common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data
    src/common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
    C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files 
(x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/encode.c /Fobuild\temp.win-amd64-3.7\Release\src/encode.obj
    encode.c
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data        
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data      
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
    src/encode.c(55): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(55): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
    src/encode.c(80): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(80): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(76): warning C4244: 'initializing': conversion from 'int' to 'float', possible loss of data
    src/encode.c(105): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(106): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(107): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data
    src/encode.c(105): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
    src/encode.c(106): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
    src/encode.c(107): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data
    C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Isrc/ "-Ic:\users\raven\desktop\code\projects\web news collecter\back\venv\include" -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include -IC:\Users\Raven\AppData\Local\Programs\Python\Python37\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files 
(x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/decode.c /Fobuild\temp.win-amd64-3.7\Release\src/decode.obj
    decode.c
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(12): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4305: 'function': truncation from 'double' to 'float'
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(13): warning C4244: 'return': conversion from 'double' to 'int', possible loss of data
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(18): warning C4244: 'return': conversion from 'double' to 'float', possible loss of data        
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4244: 'function': conversion from 'double' to 'float', possible loss of data      
    C:\Users\Raven\AppData\Local\Temp\pip-install-6_2cffe1\blurhash-python_413df5cf779e4f30a240ff18e44ce815\src\common.h(19): warning C4305: 'function': truncation from 'double' to 'float'
    src/decode.c(43): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
    src/decode.c(38): warning C4267: 'initializing': conversion from 'size_t' to 'const int', possible loss of data
    src/decode.c(57): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
    src/decode.c(58): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
    src/decode.c(71): warning C4244: 'function': conversion from 'int' to 'float', possible loss of data
    src/decode.c(82): error C2057: expected constant expression
    src/decode.c(82): error C2466: cannot allocate an array of constant size 0
    src/decode.c(82): error C2133: 'colors': unknown size
    src/decode.c(114): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.28.29910\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\raven\desktop\code\projects\web news collecter\back\venv\scripts\python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"'; __file__='"'"'C:\\Users\\Raven\\AppData\\Local\\Temp\\pip-install-6_2cffe1\\blurhash-python_413df5cf779e4f30a240ff18e44ce815\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Raven\AppData\Local\Temp\pip-record-8gga2j_2\install-record.txt' --single-version-externally-managed --compile --install-headers 'c:\users\raven\desktop\code\projects\web news collecter\back\venv\include\site\python3.7\blurhash-python' Check the logs for full command output.
```

**Comment by lautat** [comment]
Hello, and thank you for the report! It looks like the decoder implementation is using a C language feature which is not supported by MSVC, at least by default. I'll look into this and try to get it fixed soon. While waiting for the fix, you can try to use version 1.0 of this library `blurhash-python==1.0.2`, unless you need the blurhash decoder.

**Comment by lautat** [comment]
I have fixed the issue and [published version 1.1.2](https://pypi.org/project/blurhash-python/1.1.2/#files) which includes the fix. Thank you for reporting the issue!
